boards/arm/qemu_cortex_r5/qemu_cortex_r5.dts

```diff
@@ -13,10 +13,11 @@
 	compatible = "xlnx,zynqmp-qemu";
 
 	chosen {
-		zephyr,sram = &sram0;
-		zephyr,flash = &flash0;
-		zephyr,console = &uart0;
+		zephyr,sram       = &sram0;
+		zephyr,flash      = &flash0;
+		zephyr,console    = &uart0;
 		zephyr,shell-uart = &uart0;
+		zephyr,ocm        = &ocm;
```

Comment:
This must be documented in the `devicetree-chosen-nodes` section of the devicetree reference pages.

Reply:
I've added the documentation of the "zynq,ocm" chosen entry to the devicetree reference docs, comp. the newly added commit.